src/main/scala/temple/generate/server/go/service/GoServiceMainGenerator.scala

```diff
@@ -21,12 +27,28 @@ object GoServiceMainGenerator {
         //doubleQuote("fmt"),
         //doubleQuote("log"),
         //doubleQuote("net/http"),
+
+        // TODO: This check is temporary to make the integration tests pass
+        when(
+          clientAttributes
+            .exists {
+              case (_, attr) =>
+                attr.attributeType == AttributeType.DateType ||
+                attr.attributeType == AttributeType.TimeType ||
+                attr.attributeType == AttributeType.DateTimeType
```

Comment:
personally it feels like `Seq(AttributeType.DateType,AttributeType.TimeType,AttributeType.DateTimeType) contains attr.attributeType` would be nicer, but happy to be overruled

Reply:
temporary for integration tests to pass, see TODO comment in the lines above